service/arm_templates/security_groups_setup.json

```diff
@@ -148,7 +151,7 @@
               "sourcePortRange": "*",
               "destinationPortRange": "*",
               "sourceAddressPrefix": "*",
-              "destinationAddressPrefix": "[parameters('workerSubnetCidr')]",
+              "destinationAddressPrefix": "*",
```

Comment:
all changes for worker security group reflect changes for master. See comments above